sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -1325,3 +1325,82 @@ def test_pairwise_boolean_distance():
     nn1 = NN(metric="jaccard", algorithm='brute').fit(X)
     nn2 = NN(metric="jaccard", algorithm='ball_tree').fit(X)
     assert_array_equal(nn1.kneighbors(X)[0], nn2.kneighbors(X)[0])
+
+
+def test_radius_neighbors_predidct_proba():
+    for i in range(5):
+        X, y = datasets.make_classification(n_samples=50, n_features=5,
+                                            n_informative=3, n_redundant=0,
+                                            n_classes=3, random_state=i)
+        X_tr, X_te, y_tr, y_te = train_test_split(X, y, random_state=0)
+        random_label = np.random.randint(-5, 5)
+        clf = neighbors.RadiusNeighborsClassifier(radius=2,
+                                                  outlier_label=random_label)
+        clf.fit(X_tr, y_tr)
+        pred = clf.predict(X_te)
+        proba = clf.predict_proba(X_te)
+        proba_label = proba.argmax(axis=1)
+        proba_label = np.where(proba.sum(axis=1) == 0,
+                               random_label, proba_label)
+        assert_array_equal(pred, proba_label)
+
+
+def test_radius_neighbors_outliers():
```

Comment:
There are already `test_radius_neighbors_classifier_when_no_neighbors` and `test_radius_neighbors_classifier_outlier_labeling`. Please merge this into those. Thanks